tests/features/support/steps.js

```diff
@@ -210,7 +208,7 @@ When(/I click the (button|link) "([^"]+)"/, async (elemType, phrase) => {
 
   if (elem) {
     await Promise.all([
-      elem.click(),  // TODO: change to `clickOrTap`
+      elem[ scope.device_map[ scope.emulating ]](),
```

Comment:
Since we're using this `scope.device_map[ scope.emulating ]` formulation so much, it would be handy to make it its own constant that we can just refer to. Maybe something like `const interact = scope.device_map[ scope.emulating ]`?

Reply:
That's seems waaaaaay to simple and smart. Awesome! I'll put it on the project board! I might make it `scope.interact` or something so that other contexts can have access to it.